src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -2083,6 +2083,14 @@ PeerImp::getLedger (std::shared_ptr<protocol::TMGetLedger> const& m)
     p_journal_.trace <<
         "GetLeder: " << logMe;
 
+    int depth = -1;
+    if (packet.has_querydepth())
+        depth = packet.querydepth();
```

Comment:
The check immediately below will catch that case, but it's a good question: should we penalize the peer?